jdaviz/configs/default/plugins/model_fitting/initializers.py

```diff
@@ -7,21 +7,40 @@
 """
 import numpy as np
 
+import astropy.modeling.models as models
+from astropy import units as u
+from jdaviz.models import BlackBody
```

Comment:
Nitpick: Separate third-party imports from local imports.

```suggestion
from astropy import units as u

from jdaviz.models import BlackBody
```